src/applications/vaos/appointment-list/components/AppointmentsPageV2/AppointmentListItem.jsx

```diff
@@ -120,24 +121,18 @@ export default function AppointmentListItem({ appointment, facility }) {
       </div>
       <div>
         <Link
-          aria-hidden="true"
+          aria-label={`Details for ${
+            canceled ? 'canceled ' : ''
+          }appointment on ${appointmentDate.format('dddd, MMMM D h:mm a')}`}
           to={isCommunityCare ? `cc/${appointment.id}` : `va/${appointment.id}`}
           className="vads-u-display--none medium-screen:vads-u-display--inline"
         >
           Details
         </Link>
-        <Link
```

Comment:
Removing this Link makes it so that there's no link visible/clickable on mobile screen sizes. 

The original intent of this component was that the "Details" text was the link at desktop screen sizes, and at mobile screen sizes the Details text was hidden and the arrow became the link (and had a class that styled the link around the arrow such that the whole card was clickable).

Reply:
Created 2 `<div>`. The first div uses "Details" as the link and is only visible to medium screen and larger. The second div uses the arrow as the link and is only visible to small screens